cristin-commons/src/main/java/no/unit/nva/cristin/common/handler/CristinQueryHandler.java

```diff
@@ -21,7 +21,7 @@
 
     private static final Set<String> VALID_QUERY_PARAMS = Set.of(QUERY, PAGE, NUMBER_OF_RESULTS);
 
-    public static final String WORD_WHITESPACE_DASH_COMMA_PERIOD = "[\\w\\s\\-,.]+";
+    public static final String WORD_WHITESPACE_DASH_COMMA_PERIOD = "[\\w\\s\\-,.æøå]+";
```

Comment:
What do we define as a "valid query"? 

If the intention is to avoid something (like SQL injection?), then we need perhaps to look at this again.

Alternatively, any letter in any letter-based character set (includes Latin, Greek, Russian and probably others)
`^[\p{L}\x20\-,.]+$`

On the other hand, if we want any Latin letter, it becomes:

`^[A-Za-z\\u00C0-\\u00D6\\u00D8-\\u00f6\\u00f8-\\u00ff\x20\-,.]+$`

I guess it is dependent on what your match is trying to achieve and what the content is allowed to be. If it's names, then some names may. have e.g. 'ç, þ', etc.

Reply:
Having problems with numbers (for grant id search) and æøå with these regex. Maybe we should do as discussed and only check not blank?